src/renderers/WebGLRenderer.js

```diff
@@ -1857,7 +1863,62 @@ function WebGLRenderer( parameters ) {
 
 		}
 
-		state.bindFramebuffer( _gl.FRAMEBUFFER, framebuffer );
+		if ( state.bindFramebuffer( _gl.FRAMEBUFFER, framebuffer ) && capabilities.multiRenderTarget ) {
```

Comment:
Do you mind explaining a bit the purpose of this code section? Maybe by adding a few short comments?

Reply:
It sets up draw buffers to which fragment colors are written into depending on render target type.